torch/testing/_internal/common_utils.py

```diff
@@ -314,6 +314,15 @@ def run_tests(argv=UNITTEST_ARGS):
 IS_MACOS = sys.platform == "darwin"
 IS_PPC = platform.machine() == "ppc64le"
 
+def is_avx512_vnni_supported():
+    if sys.platform != 'linux':
+        return False
+    with open("/proc/cpuinfo", encoding="ascii") as f:
+        lines = f.read()
+    return "avx512vnni" in lines
```

Comment:
this feature test is not very good but I guess it'll do for now

Reply:
Yes, this code would be removed when #59098 would be fixed.